Update README.md: How to translate selected text in Wayland. (#673)
crow-translate/crow-translate: README.md

- [Default keyboard shortcuts](#default-keyboard-shortcuts)
- [CLI commands](#cli-commands)
- [D-Bus API](#d-bus-api)

> - [Translate selected text in Wayland](#how-to-translate-selected-text-in-wayland)

Reviewer: ```suggestion
- [Global shortcuts in Wayland](#global-shortcuts-in-wayland)
```

qdbus io.crow_translate.CrowTranslate /io/crow_translate/CrowTranslate/MainWindow open
```

> ## How to translate selected text in Wayland

Reviewer: ```suggestion
## Global shortcuts in Wayland
```

Wayland doesn't provide API for global shortcuts and you need to register them by yourself. 

> ### KDE

Reviewer: Use newlines after headings for consistency:
```suggestion
### KDE

```

KDE have a convenient feature to define shortcuts in .desktop file and import them in settings. These shortcuts are already enabled and should work by default.

> ### GNOME

Reviewer: ```suggestion
### GNOME

```

> For GNOME you need to use the following D-BUS command and set a hotkey:
> ```bash
> qdbus io.crow_translate.CrowTranslate /io/crow_translate/CrowTranslate/MainWindow translateSelection
> ```

Reviewer: ```suggestion
For GNOME you need to manually set D-Bus commands as global shortcuts. For example, to translate selected text use the following:

```bash
qdbus io.crow_translate.CrowTranslate /io/crow_translate/CrowTranslate/MainWindow translateSelection
```

```
